README.md

```diff
@@ -25,3 +25,30 @@ To run [jekyll](https://jekyllrb.com/) and start hacking run:
 
 
 You should see the rendered site on http://localhost:4000
+
+## PDF generation
+
+You need to have [`pandoc`](https://pandoc.org/installing.html) and LaTeX
+(`lualatex`) installed. We suggest to use
+[TinyTex](https://yihui.name/tinytex/) because it is lightweight,
+cross-platform, portable, and easy-to-maintain.
+
+The following LaTeX packages need to be installed:
+
+- `environ`
+- `everypage`
+- `fancyhdr`
+- `luatexbase`
+- `noto`
+- `pgf`
+- `selnolig`
+- `sourcecodepro`
+- `tcolorbox`
+
+The above packages could be easily installed by running `./book/deps.sh` script.
```

Comment:
would the script work on other systems that are not Linux?

Reply:
No, it would not. More than that, it won't work on any Linux other than Debian Stable.

Reply:
@pshevtsov  Do you think it is worth mentioning?